apps/theming/lib/ImageManager.php

```diff
@@ -223,7 +223,7 @@ public function updateImage(string $key, string $tmpFile) {
 			throw new \Exception('Unsupported image type');
 		}
 
-		if ($key === 'background' && strpos($detectedMimeType, 'image/svg') === false) {
+		if ($key === 'background' && (strpos($detectedMimeType, 'image/svg') === false || strpos($detectedMimeType, 'image/gif') === false)) {
```

Comment:
Sorry, I was also proposing something wrong there. It should of course be converted only if not svg AND not gif:

```suggestion
		if ($key === 'background' && strpos($detectedMimeType, 'image/svg') === false && strpos($detectedMimeType, 'image/gif') === false) {
```